stable/artifactory-ha/templates/_helpers.tpl

```diff
@@ -0,0 +1,46 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "artifactory.name" -}}
```

Comment:
Namespace should match the chart name.

Reply:
I don't understand this comment... where do you mean?

Reply:
Sorry, I may have been unclear. The chart's name is `artifactory-ha`, thus the templates should be like `{{- define "artifactory-ha.name" -}}`.